src/Functions/UserDefined/UserDefinedExecutableFunctionFactory.cpp

```diff
@@ -182,54 +184,92 @@ class UserDefinedFunction final : public IFunction
             column_with_type = std::move(column_to_cast);
         }
 
-        ColumnWithTypeAndName result(result_type, configuration.result_name);
-        Block result_block({result});
+        // catch UDF execution error and throw it as UDF_EXECUTION_FAILED
+        String args_str = Poco::cat(String(", "), command_arguments_with_parameters.begin(), command_arguments_with_parameters.end());
```

Comment:
The arguments containing whitespace or comma will be printed in a confusing way.